### DISCUSSION TITLE
SSD1302

### DISCUSSION THREAD

**Discussion by amaurylll03** [olikraus/u8g2#2049] [Q&A] [answered]
I am working on a project with an [SSD1306] and we are happy to use your library.But unfortunatly we are working on an arduino nano so there are some problem
[aaa.zip](https://github.com/olikraus/u8g2/files/10220490/aaa.zip)
 onRAM memory.Do you think it is possible to change your librarie into an only SSD1306 librarie?

**Comment by olikraus** [comment, marked as answer]
It is possible, but there would be no reduction in memory usage. U8g2 makes heavy use of LTO (link time optimization) and all code which is not needed for your display is automatically removed. If you run out of memory, you either need a bigger uC or you need to optimize your code. For example you should use the page mode in cases where you use the full buffer mode.
